### PR TITLE
[FIX] point_of_sale : show correctly UserError

### PR DESCRIPTION
**Pull request by fmdl**
**Description of the issue/feature this PR addresses:**
Before this commit UserError, ValidationError are not correctly show.

**Current behavior before PR:**
image

**Desired behavior after PR is merged:**
image

@pimodoo @rhe-odoo 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fmdl** [comment]
ping @caburj

**Comment by caburj** [comment]
@fmdl It would be nice if the `title` of the popup is the "UserError". Can we recover that from the error?

**Comment by fmdl** [comment]
I have use the same like here for error:
image

it is possible to use error.message (Odoo User Error), but it is not translated.

File: addons/point_of_sale/static/src/js/Chrome.js [hunk 1]
```diff
@@ -385,6 +385,18 @@ odoo.define('point_of_sale.Chrome', function(require) {
         _replaceCrashmanager() {
             var self = this;
             CrashManager.include({
+                show_warning: function (error) {
+                    if (self.env.pos) {
+                        // self == this component
+                        self.showPopup('ErrorPopup', {
+                            title: error.message,
```
Comment (caburj) [inline addons/point_of_sale/static/src/js/Chrome.js:392]:
```suggestion
                            title: error.data.title.toString(),
```
I'm tested this and I believe we can do this suggestion.

I looked into the `CrashManager` code and it seems that the field `title` the result of `_lt`. So calling the `toString` will actually return the translated value. What do you think?

Comment (fmdl) [reply addons/point_of_sale/static/src/js/Chrome.js]: @done !

**Comment by caburj** [comment]
Hi @pimodoo. This needs your approval to proceed.

The good thing about this PR is that when there is a `UserError` from the server (or any warning error), instead of showing an unformatted modal from the web module, the pos `ErrorPopup` is shown. I tested this and it works well. I'll leave the final decision to you. :)

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**